api-reference/10 UI Widgets/dxMap/1 Configuration/apiKey/apiKey.md

```diff
@@ -5,7 +5,122 @@ default: { bing: '', google: '', googleStatic: '' }
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A key to authenticate the component within a map provider.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+If you have more than one map provider in your application, specify the keys in the object fields. Otherwise, specify the key as a string.
```

Comment:
- Added "as shown below" to connect the description with the code sample.
- "specify the key as a string" - multiple keys are specified as strings too, since they are strings.

```suggestion
If you have more than one map provider in your application, specify the keys in the object fields as shown below. Otherwise, you can assign the key directly to the **apiKey** property.
```